source/tyr/tyr/helper.py

```diff
@@ -348,3 +350,21 @@ def hide_domain(email):
     if "@" not in email:
         return email
     return "{}******".format(email.split('@')[0])
+
+
+def repost_to_another_tyr(logger, url, content, instance_name):
+    content.seek(0)
+    file_to_post = {"file": (content.filename, content.stream, content.mimetype)}
+    url_secondary_tyr = '{}/jobs/{}'.format(url, instance_name)
+    nb_try = 0
+    while nb_try < MAX_NB_CALLS:
+        try:
+            resp = requests.post(url_secondary_tyr, files=file_to_post, verify=False)
```

Comment:
Now that you validated that it works, should we remove this `verify=False` and try to avoid bypassing security rules?

Reply:
After installing certificate, it didn't work without this dirty fix this function will live for few weeks.
I am ready to test in dev without this fix.